cilantro_audit/widgets/view_audit_templates.kv

```diff
@@ -101,3 +101,17 @@
 <AuditButton@Button>:
     name: 'button_template'
     size_hint_y: None
+
+<InactiveAuditButton@Button>:
+    name: 'inactive_button_template'
+    size_hint_y: None
+    background_normal: ''
+    background_color: GREY_LIGHT
+    color: 0, 0, 0, 1
+
+<LockedTemplatePop>:
```

Comment:
An exit button for popups would be really nice, to keep with our popup structure of having them.